crates/bevy_asset/src/server/info.rs

```diff
@@ -181,7 +182,11 @@ impl AssetInfos {
             meta_transform,
         );
         // it is ok to unwrap because TypeId was specified above
```

Comment:
This comment is in the wrong spot. Alternatively, maybe just making this an expect is appropriate?

Reply:
Changed it to an expect, since that's why the stdlib even has that function in the first place